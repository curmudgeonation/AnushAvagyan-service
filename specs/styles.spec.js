import React from 'react';
import renderer from 'react-test-renderer';
import { shallow, mount, render } from 'enzyme';
import {
  Bold, Info, ContactHost, Note, NoteIcon, Cols, StatsData, Date, Title, Name, Avatar, Top, Image, Icon, ReadMore
} from '../client/src/components/styledComponents'
const sampleData = require('../database/sampleData.js');


describe('Styles - React Styled Components', () => {
  it('Info block renders correctly', () => {
    const tree = renderer
    .create(<Info></Info>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  });
  it('Note Icon renders correctly', () => {
    const tree = renderer
    .create(<NoteIcon></NoteIcon>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  });
  it('Note renders correctly', () => {
    const tree = renderer
    .create(<Note></Note>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  });
  it('Stats columns renders correctly', () => {
    const tree = renderer
    .create(<Cols></Cols>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  });
  it('StatsData renders correctly', () => {
    const tree = renderer
    .create(<StatsData></StatsData>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  });
  it('Top component renders correctly', () => {
    const tree = renderer
    .create(<Top></Top>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  });
  it('Contact Host button renders correctly', () => {
    const tree = renderer
    .create(<ContactHost></ContactHost>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  });
  it('Bold text renders correctly', () => {
    const tree = renderer
    .create(<Bold>During your stay</Bold>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  });
  it('Avatar renders correctly', () => {
    const tree = renderer
    .create(<Avatar></Avatar>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  });
  it('Avatar image renders correctly', () => {
    const tree = renderer
    .create(<Image></Image>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  });
  it('Superhost icon on avatar renders correctly', () => {
    const tree = renderer
    .create(<Icon></Icon>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  });
  it('Title joined renders correctly', () => {
    const tree = renderer
    .create(<Title></Title>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  });
  it('Date joined renders correctly', () => {
    const tree = renderer
    .create(<Date></Date>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  });
  it('Name renders correctly', () => {
    const tree = renderer
    .create(<Name></Name>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  });
  it('Read more button renders correctly', () => {
    const tree = renderer
    .create(<ReadMore></ReadMore>)
    .toJSON();
  expect(tree).toMatchSnapshot();
  });
});



