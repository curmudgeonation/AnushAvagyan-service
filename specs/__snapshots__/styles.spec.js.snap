// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styles - React Styled Components Avatar image renders correctly 1`] = `
<img
  className="sc-AxmLO enZMVm"
/>
`;

exports[`Styles - React Styled Components Avatar renders correctly 1`] = `
<div
  className="sc-Axmtr jIlkoi"
/>
`;

exports[`Styles - React Styled Components Bold text renders correctly 1`] = `
<div
  className="sc-AxheI iQSWCA"
>
  During your stay
</div>
`;

exports[`Styles - React Styled Components Contact Host button renders correctly 1`] = `
<button
  className="sc-AxgMl jCyJLy"
/>
`;

exports[`Styles - React Styled Components Date joined renders correctly 1`] = `
<div
  className="sc-fzpans jLqwke"
/>
`;

exports[`Styles - React Styled Components Info block renders correctly 1`] = `
<div
  className="sc-AxjAm ibykXD"
/>
`;

exports[`Styles - React Styled Components Name renders correctly 1`] = `
<h2
  className="sc-fzplWN gNuZES"
/>
`;

exports[`Styles - React Styled Components Note Icon renders correctly 1`] = `
<img
  className="sc-AxirZ SVkwc"
/>
`;

exports[`Styles - React Styled Components Note renders correctly 1`] = `
<div
  className="sc-fznKkj hTLTnj"
/>
`;

exports[`Styles - React Styled Components Read more button renders correctly 1`] = `
<a
  className="sc-fznyAO iJQZdx"
/>
`;

exports[`Styles - React Styled Components Stats columns renders correctly 1`] = `
<div
  className="sc-AxiKw dgPOlS"
/>
`;

exports[`Styles - React Styled Components StatsData renders correctly 1`] = `
<div
  className="sc-AxhCb dujZPi"
/>
`;

exports[`Styles - React Styled Components Superhost icon on avatar renders correctly 1`] = `
<img
  className="sc-fzozJi dRCMDn"
/>
`;

exports[`Styles - React Styled Components Title joined renders correctly 1`] = `
<div
  className="sc-fzoLsD inWKVQ"
/>
`;

exports[`Styles - React Styled Components Top component renders correctly 1`] = `
<div
  className="sc-AxhUy kTSBDK"
/>
`;
